conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
I also think current pluggable metrics consumer design is too flexible. Users rely on metrics consumers which are open source from individual or specific vendor, which is easy to be not maintained. I guess most popular implementation of metrics consumer is verisign/storm-graphite, and it's not maintained for months. We should provide stable consumer implementation. If we don't want to handle various time-series DBs, we can open at most pluggable connector interface.